test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -201,3 +201,25 @@ Describe "Validate about_help.txt under culture specific folder works" -Tags @('
         $help | Should BeExactly "Hello"
     }
 }
+
+
+Describe "Get-Help should find help info within help files" -Tags @('CI', 'RequireAdminOnWindows') {
+    It "Get-Help should find help files under pshome" {
+        $helpFile = "about_testCase.help.txt"
+        if (!(Test-Path $PSHOME\en-US))
+        {
+            New-Item -ItemType Directory -Path "$PSHOME\en-US" -ErrorAction SilentlyContinue
```

Comment:
Should be as follows to avoid output on console. 
```Powershell
$null = New-Item -ItemType Directory -Path "$PSHOME\en-US" -ErrorAction SilentlyContinue 

```